rules.txt

```diff
@@ -1,12 +1,215 @@
+    action: remove
+    message: |
+        Your [{{kind}}]({{permalink}}) in /r/{{subreddit}} was automatically removed because you used a URL shortener.
+
+        URL shorteners are not permitted in /r/{{subreddit}} as they impair our ability to enforce link blacklists.
+
+        Please re-post your {{kind}} using direct, full-length URL's only.
+---
+##----------------------------
+##  Spam Link Shortener Rules - Auto remove w/o comment
+##----------------------------
+domain+body: [lynk.my]
+action: remove
+---
+##----------------------------
+## New Account Rules - 1 day removal
+##----------------------------
+    author:
+       account_age: < 1
+    type: submission
+    action: remove
+    comment:  |
+        Your account is too young. Please wait at least 5 days to begin posting. You're still welcome to comment on other posts
+    action_reason: Account age
+---
+##----------------------------
+## New Account Rules - 5 day filter
+##----------------------------
+    author:
+       account_age: < 5
+    type: submission
+    action: filter
+    action_reason: Account age
+---
+##----------------------------
+## Low Karma Rules - 5 day filter
+##----------------------------
+#TODO
+---
+##----------------------------
+## Low Quality Phrase Rules
+##----------------------------
+#TODO
+---
+##----------------------------
+## Meme Rules
+##----------------------------
+#TODO
+---
+##----------------------------
+## Stallman Rules
+##----------------------------
+#TODO, an automod response rule about the what you're referring to is GNU etc
+---
+##----------------------------
+## Media Rules
+##----------------------------
+#TODO - Mark and flair various media
+---
+##----------------------------
+## Automark NSFW/L - (mostly from automoderator example)
+##----------------------------
+    title+body (regex): ["nsf[wl]", "not safe for (work|life)"]
+    action: report
+    report_reason: "{{match-1}}"
+    set_nsfw: true
+---
+##----------------------------
+## Questions indentifier rule - Send to /r/linuxquestions
+##----------------------------
+    title (ends-with): ['?']
+    action: remove
+    comment: |
+        Your [{{kind}}]({{permalink}}) in /r/{{subreddit}} was automatically removed for being a question.
+
+        /r/{{subreddit}} is not a support forum, however we encourage the post to be made over on /r/linuxquestions
+---
+##----------------------------
+## Social Media Blacklist
+##----------------------------
+    domain: ['facebook.com']
+    action: remove
+    set_flair: ["Removed","removed"]
+    comment: |
+        Your [{{kind}}]({{permalink}}) in /r/{{subreddit}} was automatically removed because it is a link to non-technical social media.
+---
+##----------------------------
+## Spam-blog rules
+## Keep in mind there's various other sections about project-specific whitelisting
+##----------------------------
+    domain: ["phoronix.com", "omgubuntu.co.uk"]
```

Comment:
Could we please not mark Phoronix as spam?  Some of what they do is just quick links to other blogs, but there's a lot of really good value in their benchmarking.

Reply:
OMG Ubuntu isn't blogspam. This seems to be motivated by that hurt individual [user/magikarpkult](https://www.reddit.com/user/magikarpkult) that claims he was banned on OMG Ubuntu and now goes around all over reddit trying to ban the site everywhere. He was in contact with r/Linux about it as well and succeeded with his trolling, or at least that's what he says.

Blogspam would be characterized by a site that just aggregate other sites. OMG Ubuntu isn't in that category, the site has much original content that isn't found elsewhere. The owner is trying to stay ahead of other sites and refuses to cover stories that are already covered everywhere else.

The reason the owner gave for not writing much about KDE yet:
> I can't cover everything I want to sadly! It's usually the case that by the time I hear about a KDE release or change it was 5 days ago and every other blog has written all there is to write about it.
> KDE developers don't tell us about new releases or keep us in the loop in the way that developers of other desktops top. The easier that devs make it to write about their project, the more likely it is to get coverage.
([Source](https://www.omgubuntu.co.uk/2018/03/gnome-3-30-release-date#comment-3824892081))

The reason he gave for not writing about the Krita 4.0 release:
> Krita's PR team didn't ping us about the release, so by the time I knew it was out I was too late; every other site that covers Linux had already posted about it. Not much point in repeating what most people already know about.
([Source](https://www.omgubuntu.co.uk/2018/03/gnome-3-30-release-date#comment-3824892081))

There's no reason to ban OMG Ubuntu altogether. This can be handled on a case by case if any unoriginal posts shows up.

Reply:
The reason OMGUbuntu was marked as blogspam is because we'd rather have the official release (for instance, from KDE's blog or Ubuntu's official blog) than an article written by a 3rd party.

But, we can leave that to a vote.

Reply:
Agree with your reasoning, but content originating from a official blog isn't always telling everything there is. Often details will be spread over several posts or other mediums making it hard for people who are not already closely follows the projects to get a understanding of what's going on. Some projects don't even have a medium that they actively use, this is specially true for the smaller projects.

Voting would be appreciated. I think a voting on r/Linux would be the right thing to do for any site that are not obviously spam, as there are probably details here that should be discussed before banning.

Reply:
Would it be more prudent to have the OP be the original source, with allowing Phoronix and such in the comments?  That way, you get the official announcement, and gives a reason to visit the comment section and participate?

Reply:
@LE3M - /u/magikarpkult comments did not go into my decision to include it here, really I ignored that users comments. The fact that it is blogspam is why it was included here (and I understand it, OMGUbuntu was banned already as well). r/linux is not r/Ubuntu; if we were talking about r/Ubuntu rules then maybe we'd have more of a discussion right now for allowing it. And yes I know they do more than just Ubuntu, but its still Ubuntu focused.

We should go the route of the main link being the original submission on /r/linux. These rules don't prevent the comments from posting links to OMGUbuntu etc sometimes they can add more context for a discussion. That's what the comments are for, and if the different discussions are in the comments of the official source then we avoid the issue of multiple posts containing the discussions.

Re: Voting on rules - I moderate other subreddits and voting on rules never works out. The most vocal users are not always right. For me, being able to submit the rules on GitHub is how I'm helping shape /r/Linux.

Reply:
As pointed out through other discussions, when Phoronix was unbanned for posts, /r/Linux basically became /r/Phoronix.  Same happened with OMGUbuntu.

Voting on rules won't be a "Hey, they voted this way so it's final."  It's more of getting a pulse on the community.  The rules may be influenced by the results, but it won't be fully democratic.